music_assistant/providers/sonos/provider.py

```diff
@@ -67,6 +69,24 @@ async def handle_async_init(self) -> None:
             "/sonos_queue/v2.3/timePlayed", self._handle_sonos_queue_time_played
         )
 
+        ips = self.config.get_value(CONF_IPS).split(",")
+        for raw_ip in ips:
+            ip = raw_ip.strip()
+            if ip == "":
+                continue
+            try:
+                discovery_info = await get_discovery_info(self.mass.http_session, ip)
+            except ClientError as err:
+                self.logger.debug(
+                    "Ignoring %s in static IP as it is not reachable: %s", ip, str(err)
+                )
+                continue
+            player_id = discovery_info["device"]["id"]
+            self.sonos_players[player_id] = sonos_player = SonosPlayer(
+                self, player_id, discovery_info=discovery_info, ip_address=ip
+            )
+            self.mass.call_later(5, sonos_player.setup, task_id=f"setup sonos {ip}")
+
```

Comment:
Please move this entire block out of the init and into "loaded_in_mass". Also add a bit of comments what it does. You can also do the setup directly and not later